### ISSUE TITLE
Buy American, Hire American - which promise does this fit under? [ACHIEVED]

### ISSUE THREAD

**Issue by VirenMohindra** [TrumpTracker/trumptracker.github.io#138]
### Description of your *issue*, suggested solution and other information

Might need to create a new promise and mark it as achieved. Thoughts?

https://www.whitehouse.gov/the-press-office/2017/04/18/presidential-executive-order-buy-american-and-hire-american
---

### Make sure you are using the *latest* version: pull latest commits before reporting any issues.
- [x] I've **verified** and **I assure** that I'm looking at the latest version of Trump Tracker.

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/TrumpTracker/trumptracker.github.io/blob/master/README.md) and **most notably** the [To Do List](https://github.com/TrumpTracker/trumptracker.github.io#to-do-list) section.
- [x] [Searched](https://github.com/TrumpTracker/trumptracker.github.io/search?type=Issues) the bugtracker for similar issues including closed ones

**Comment by iProdigy** [comment]
I would mark this promise as `compromised` rather than `achieved` as Trump consistently promised that the Keystone XL pipeline would be built with American steel but [that's not happening](http://money.cnn.com/2017/03/03/news/companies/trump-keystone-pipeline-american-steel/).

**Comment by VirenMohindra** [comment]
Good catch -- since the pipeline isn't government owned and the pipeline was already in progress does it still fit into this specific 'promise'? He might have promised to use American steel, but it seems a bit harsh to hold him accountable if the project was in progress before his EO.

**Comment by iProdigy** [comment]
In my view this should be included under this promise. Trump repeatedly said the Keystone XL pipeline would be built with American steel and [he is the one who authorized its construction](http://www.washingtonexaminer.com/trump-signs-executive-orders-authorizing-keystone-xl-dakota-access-pipelines/article/2612804) through a [memorandum](https://www.whitehouse.gov/the-press-office/2017/01/24/presidential-memorandum-regarding-construction-keystone-xl-pipeline) (after [Obama rejected it](https://obamawhitehouse.archives.gov/the-press-office/2015/11/06/statement-president-keystone-xl-pipeline) during his term).

**Comment by VirenMohindra** [comment]
Strong points to consider. Lets get more people to weigh in on this before making a decision.

**Comment by VirenMohindra** [comment]
I can safely assume this promise aligns with his 

> s. 26 '...reforms visa rules to enhance penalties for overstaying and to ensure open jobs are offered to American workers first.'

which is already marked as 'in Progress'. Best we can do is add these sources to the promise, what do you think?

**Comment by iProdigy** [comment]
I don't know if it makes sense to add this action to that specific promise as it is in reference to getting the End Illegal Immigration Act passed.

**Comment by VirenMohindra** [comment]
Should we separate the promises?
